tools/user/mergeq/mergeq.js

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env node
+import fetch from "node-fetch";
+
+// Replace with your details
+const owner = "enterprisedb";
+const repo = "docs";
+const token = ""; /*process.env.GITHUB_TOKEN*/ // Optional - if we ever need to make authenticated requests
+
+async function getLatestReleaseDate(owner, repo, token) {
+  const url = `https://api.github.com/repos/${owner}/${repo}/releases/latest`;
+  const headers = token ? { Authorization: `token ${token}` } : {};
+  const response = await fetch(url, { headers });
+  if (!response.ok) {
+    throw new Error(`Error fetching latest release: ${response.statusText}`);
+  }
+  const data = await response.json();
+  return new Date(data.published_at);
```

Comment:
just a note for future us: this opens the door for missing PRs merged between the creation of the release branch and the publishing of the Release.